package.json

```diff
@@ -13,6 +13,8 @@
     "heroku-postbuild": "npm install",
     "migration:generate": "node_modules/.bin/sequelize migrations:generate --name",
     "migrate": "node_modules/.bin/sequelize db:migrate",
+    "seed": "NODE_ENV=test node_modules/.bin/sequelize db:seed:all",
+    "seed:travis": "node_modules/.bin/sequelize db:seed:all",
```

Comment:
Don't repeat yourself. By default Travis environment is tested, you can run the same seed script instead of creating two.

Reply:
Correct! I had forgotten that we are using the test environment on TravisCI too